src/main/java/net/neoforged/neoforge/common/CommonHooks.java

```diff
@@ -921,22 +919,6 @@ public static boolean canEntityDestroy(Level level, BlockPos pos, LivingEntity e
         return EventHooks.getMobGriefingEvent(level, entity) && state.canEntityDestroy(level, pos, entity) && EventHooks.onEntityDestroyBlock(entity, pos, state);
     }
 
-    /**
-     * Gets the burn time of this itemstack.
-     */
-    public static int getBurnTime(ItemStack stack, @Nullable RecipeType<?> recipeType) {
```

Comment:
Are we still in beta? If not, this needs to be deprecated instead.

Reply:
Since 1.20.5 may be delayed or cancelled, it is reasonable to move this to a deprecation and perform this as non-breaking.